tests/data/src/requires_capitalized/setup.py

```diff
@@ -0,0 +1,6 @@
+from setuptools import find_packages, setup
```

Comment:
`find_packages` isn't used

Reply:
I left it there because it appeared in the original file tests/data/src/requires_simple/setup.py and I didn't want to cause any weird import vodoos.
Should I remove the import from both files?

Reply:
Don't be afraid of voodoo. If removing an unused import broke something and there wasn't a comment above it saying why, then that's a problem we would want to know about!

Generally speaking it's a good idea to only clean up code that you're already touching. It can vary by project how much people care. In pip I think we care a medium amount, mostly motivated by keeping PRs easier to evaluate. So in this case I would just remove it from your new `setup.py`.